docs/development/build-instructions-linux.md

```diff
@@ -73,7 +73,14 @@ Downloading certain files can take a long time. Notice that we are using
 
 ```sh
 $ cd electron
-$ ./script/bootstrap.py --verbose
+$ npm run bootstrap
+```
+
+You can also bootstrap for debug builds:
```

Comment:
Moot if the above suggestion is taken, but this should be `developer` not `debug`